code/analyticsedge/src/phone/java/com/adobe/marketing/mobile/AnalyticsExtension.java

```diff
@@ -192,6 +215,65 @@ public void handleAnalyticsTrackEvent(final Event event) {
         }
     }
 
+    /**
+     * Adds the Rules Engine Response Content event to the event queue to be processed.
+     *
+     * <p>
+     * Any events in the event queue will be cleared if the privacy status is opted out.
+     *
+     * @param event The Rules Engine Response Content {@link Event} to be processed.
+     */
+    @Override
+    public void handleRulesEngineEvent(final Event event) {
+        if (event == null) {
+            Log.debug(AnalyticsConstants.LOG_TAG, "Unable to handle rules engine response. Event received is null.");
+            return;
+        }
+
+        eventQueue.add(event);
+        processEvents();
+    }
+
+    /**
+     * Processes the passed in Rules Engine Response Content event.
+     *
+     * @param event The Rules Engine Response Content {@link Event} to be processed.
+     * @param data Track data for processing
+     */
+    private void processRulesEngineResponseEvent(final Event event, final Map<String, Object> data) {
+        if(data == null || data.isEmpty()) {
+            Log.trace(LOG_TAG, "processRulesEngineResponseEvent - Event with id %s contained no data, ignoring.", event.getUniqueIdentifier());
```

Comment:
Can we make it debug.

Reply:
i matched this to the iOS implementation. i think debug should be fine as this log shouldn't be triggered too often.